bot/extensions/fun/fun_cog.py

```diff
@@ -39,6 +41,20 @@ async def quote_command(self, ctx):
 
         await ctx.send(embed=embed)
 
+    @command(name='rgb', help='''Displays the RGB color entered by the user.''')
+    async def color_command(self, ctx, R: int, G: int, B: int):
+        rgb = (R, G, B)
+        img = Image.new('RGB', (200, 200), rgb)
+
+        img.save('color.png')
+        embed = Embed(
+            color=self.bot.default_color,
+            description='Here goes your color!'
+        )
+        embed.set_image(url="attachment://filename.png")
+        file = File('color.png')
```

Comment:
Same thing here, I don't know if it's really necessary to assign it to `file` but it's optional. I don't mind if it's stay like this. :)

Reply:
I think it's good practice to initialize objects separately maybe.
Debatable, but maybe it'll stay like this :P

Reply:
I wouldn't say it's "good" practice exactly but it's for sure isn't bad so it's fine.